ghcide/test/ghcide-test-utils.cabal

```diff
@@ -3,7 +3,7 @@ cabal-version:      3.0
 build-type:         Simple
 category:           Development
 name:               ghcide-test-utils
-version:            1.9.0.0
+version:            2.0.0.0
```

Comment:
Nitpick: should we update tested-with field to reflect what's being tested in CI?